Exercise_D and E/assets/style/index.css

```diff
@@ -1,4 +1,77 @@
-/* 
-This css sheet is linked to practice.html
-You can style tags, classes (.) or ids (#) from this file.
-*/
+body{
+    box-sizing: border-box;
+    margin:0;
+}
+
```

Comment:
Try to stick to one colour convention: If you start by using RGBA continue using it within your code.